include/openPMD/WriteIterations.hpp

```diff
@@ -59,7 +65,6 @@ class WriteIterations : private Container< Iteration, uint64_t >
         ~SharedResources();
     };
 
-    using key_type = typename iterations_t::key_type;
     using value_type = typename iterations_t::key_type;
```

Comment:
Agreed, I think that's a copy-paste error:
```suggestion
    using value_type = typename iterations_t::value_type;
```

Reply:
Since that type definition is unused and private, I would even remove it for now:
```suggestion
```